test/unit/any_instance_method_test.rb

```diff
@@ -18,14 +25,14 @@ def test_should_hide_original_method
   end
 
   def test_should_not_raise_error_hiding_method_that_isnt_defined
-    klass = Class.new
+    klass = class_with_method(nil)
```

Comment:
I think this test is now a bit confusing, because in this case `klass` is actually a class *without* a method! It might be better to have a `new_class` method which yields a block and just replaces `Class.new`.

Reply:
Resolving - the whole branch has been re-written.